src/app/modules/consultations/consultation-profile/read-respond/read-respond.component.html

```diff
@@ -5,33 +5,40 @@
       <span>{{profileData?.readingTime ? '(' + profileData?.readingTime + ' min)' : '' }}</span>
     </p>
     <div *ngIf="profileData?.page || profileData?.summaryHindi">
-      <app-page-content-renderer [page]="useSummaryHindi ? profileData?.summaryHindi : profileData?.page"></app-page-content-renderer>
+      <app-page-content-renderer [page]="useSummaryHindi ? profileData?.summaryHindi : profileData?.page">
+      </app-page-content-renderer>
     </div>
   </div>
 
-  <div class="paper-effect mt-4 m-0-sm" *ngIf="showCreateResponse() && questionnaireExist() && !showQuestions"
-    #startDraftingSection>
-    <div class="draft-box">
-      <div class="draft-img">
-        <img src="assets/images/start-drafting-empty-state.svg" />
-      </div>
-      <p class="draft-text">
-        Ready to draft your response? Click <br /> the below button to start
-      </p>
-      <button  class="btn-submit  active" (click)='showQuestions = true'>
-        <p class="m-0">
-          START DRAFTING
-        </p>
-        <span class="ml-2"><i class="fa fa-angle-right"></i></span>
-      </button>
+<section #questionnaireContainer>
+  <div class="paper-effect  mt-4 m-0-sm" *ngIf="showCreateResponse() && questionnaireExist()">
+    <p class="satisfactory-heading">
+      {{'How satisfied are you with this Consultation' | translate}}
+    </p>
+    <p class="satisfactory-tip">
+      {{'Choose an option to submit your feedback' | translate}}
+    </p>
+    <div class="options-block">
+      <button 
```

Comment:
@Nishanthh Emojis are missing for the options

Reply:
@Nishanthh Emojis are added?

Reply:
No No bro, doing that.